internal/http/services/owncloud/ocs/data/capabilities.go

```diff
@@ -110,6 +126,8 @@ type CapabilitiesFiles struct {
 	Favorites        ocsBool                      `json:"favorites" xml:"favorites"`
 	BlacklistedFiles []string                     `json:"blacklisted_files" xml:"blacklisted_files>element" mapstructure:"blacklisted_files"`
 	TusSupport       *CapabilitiesFilesTusSupport `json:"tus_support" xml:"tus_support" mapstructure:"tus_support"`
+	Archivers        []*CapabilitiesArchiver      `json:"archivers" xml:"archivers" mapstructure:"archivers"`
```

Comment:
If there are multiple archiver or app provider versions available from the backend, how do I know which one is the most recent one? By parsing the versions? Or do you think it makes sense to introduce some `MostRecent` boolean in the respective capability struct?